applications/GeoMechanicsApplication/tests/cpp_tests/test_convergence_criteria_factory.cpp

```diff
@@ -0,0 +1,57 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "testing/testing.h"
+#include "custom_utilities/convergence_criteria_factory.hpp"
+#include "spaces/ublas_space.h"
+#include "solving_strategies/convergencecriterias/mixed_generic_criteria.h"
+
+const std::string validParameters = R"(
+    {
+        "convergence_criterion":              "displacement_criterion",
+        "displacement_relative_tolerance":    1.0E-4,
+        "displacement_absolute_tolerance":    1.0E-9
+    }
+)";
```

Comment:
I would move this to the body of the single test case where these settings are being used.

Reply:
Agreed, I thought I would need to re-use it, but that's not the case